### ISSUE TITLE
[Navitia] NotFoundException: fr-sw/stop_point

### ISSUE THREAD

**Issue by grote** [schildbach/public-transport-enabler#59]
When trying to find nearby stations for "Bardanac" I am getting this exception:
```
de.schildbach.pte.exception.NotFoundException: http://api.navitia.io/v1/coverage/fr-sw/stop_point/stop_area:O33:SA:19727/places_nearby?type[]=stop_point&distance=0&count=5&depth=0
     at de.schildbach.pte.util.HttpClient.getInputStream(HttpClient.java:228) 
     at de.schildbach.pte.util.HttpClient.get(HttpClient.java:105)
     at de.schildbach.pte.util.HttpClient.get(HttpClient.java:95)
     at de.schildbach.pte.util.HttpClient.get(HttpClient.java:90)
     at de.schildbach.pte.AbstractNavitiaProvider.queryNearbyLocations(AbstractNavitiaProvider.java:795)
```
@kalon33 or @aelkhour might want to look at this one since it happens with Navitia and the newly added FR-SW region.

**Comment by grote** [comment]
This happens also with the recently added Italy provider. Maybe the API endpoint has changed?

**Comment by kalon33** [comment]
This happens also with Paris, in both Offi and Transportr apps, that use this library. So yes it's probably linked to some change in Navitia API.

**Comment by kalon33** [comment]
@grote @schildbach do you have some way to quickly test that fix ( #61 ) ? I ran into problems on my machine that is very slow to set up a proper test environment.
